sdk/iot/Azure.Iot.Hub.Service/tests/Azure.Iot.Hub.Service.Tests.csproj

```diff
@@ -9,6 +9,7 @@
     <!--Temporarily using a track 1 library for testing this track 2 service client. Once
     the device client is track 2 as well, remove this dependency-->
     <PackageReference Include="Microsoft.Azure.Devices.Client" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" />
```

Comment:
It was a todo to try with track2 blob storage.

Reply:
I'm planning to remove this altogether in the next PR. We will create the blob and the SAS token in our script as the core framework cannot mock this.